packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -273,6 +274,84 @@ export abstract class RestApiBase extends Resource implements IRestApi {
     return this._domainName;
   }
 
+  /**
+   * Return the given named metric for this API
+   */
+  public static metricAll(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return new cloudwatch.Metric({
+      namespace: 'AWS/ApiGateway',
+      metricName,
+      ...props,
+    });
+  }
+
+  /**
+   * Metric for the number of client-side errors captured in a given period.
+   *
+   * @default sum over 5 minutes
```

Comment:
```suggestion
   * @default - sum over 5 minutes
```

same fix in other places.